qutebrowser/config/configdata.yml

```diff
@@ -88,6 +91,13 @@ session_default_name:
 
     If this is set to null, the session which was last loaded is saved.
 
+session.lazy_restore:
+  type:
+    name: Bool
+    none_ok: true
```

Comment:
This means that you can set `session.lazy_restore = None` (and not only `True` or `False`) - I don't think this makes sense here?

Note that when you remove it, you can just do `type: Bool` without the additional `name: ...`.